lib/pact_broker/pact_merger.rb

```diff
@@ -0,0 +1,56 @@
+require 'json'
+
+module PactBroker
+  module PactMerger
+
+    extend self
+
+    # Accepts two hashes representing pacts, outputs a merged hash
+    # Does not make any guarantees about order of interactions
+    def merge_pacts original_json, additional_json
+      original, additional = [original_json, additional_json].map{|str| JSON.parse(str) }
```

Comment:
Can you do a search for the usage of PACT_PARSING_OPTIONS, note how it is used, and add it here? Otherwise, the parsing will explode when the JSON is nested too deeply.